docs/pages/submit/android.mdx

```diff
@@ -11,7 +11,7 @@ This guide outlines how to submit your app to the Google Play Store from your ow
 ## Prerequisites
 
 - A paid developer account is required &mdash; You can create a Google Play Developer account on the [Google Play Console sign-up page](https://play.google.com/apps/publish/signup/).
-- You have to create a [Google Service Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) and download its JSON private key.
+- You have to create a [Google Service Account](#creating-a-google-service-account) and download its JSON private key.
```

Comment:
```suggestion
- You have to create a [Google Service Account](https://expo.fyi/creating-google-service-account) and download its JSON private key.
```

That's a great suggestion! Yeah, we can link to the FYI guide instead of linking the section and then people will find the link to the guide once again in that section.

I've updated initial suggested to include the FYI guide.